scripts/pipeline/create_all_mlss.pl

```diff
@@ -179,10 +179,11 @@ =head2 BEHAVIOUR CONFIGURATION
 my @mlsss;
 
 sub find_genome_from_xml_node_attribute {
-    my ($xml_node, $attribute_name) = @_;
+    my ($xml_node, $attribute_name, $allow_old) = @_;
+    $allow_old //= 0;
     my $species_name = $xml_node->getAttribute($attribute_name);
-    my $gdb = $genome_dba->fetch_by_name_assembly($species_name) || throw("Cannot find $species_name in the available list of GenomeDBs");
-    die "Cannot find any current genomes with species_name '$species_name'. Please check that this name is still correct" unless $gdb->is_current;
+    my $gdb = $genome_dba->fetch_by_name_assembly($species_name) || $genome_dba->fetch_by_dbID($species_name) || throw("Cannot find $species_name in the available list of GenomeDBs");
+    die "Cannot find any current genomes matching '$species_name'. Please check that this name is still correct" unless ($allow_old || $gdb->is_current);
```

Comment:
I may be misreading this, but I am not sure this will behave as you would expect, it may never die because `$allow_old` has been initialised? Fine if it has been tested and proven otherwise, will take your word for it!

Reply:
If `$allow_old` is 0 (default), then `$gdb->is_current` is what decides if it dies or not, and when `$allow_old` is 1, it will never die.